index.html

```diff
@@ -2098,6 +2098,17 @@ <h4>Semantic Interoperability</h4>
             </li>
           </ul>
 
+          <p>
+If implementers follow the requirements above, they are guaranteed semantic
+interoperability between JSON and JSON-LD syntaxes for terms defined by the
+<code>@context</code> mechanism. This is because the exact terms that a
+producer and consumer of the data model described by this specification,
+as well as any extensions used, are explicitly stated through the use and
+processing of the <code>@context</code> property. In other words, the context
```

Comment:
I had trouble parsing the sentence starting with `This is because...` ... I think a word was forgotten or something?

Reply:
Fixed in ad5c9bcd1b2b956fcb0ccc7749d8a2915e1b8fd5.